XmlFormatter/src/Windows/MainForm.cs

```diff
@@ -567,11 +602,29 @@ private void MI_HideToTray_Click(object sender, EventArgs e)
         /// <param name="e">The arguments provided by the sender</param>
         private void MI_Settings_Click(object sender, EventArgs e)
         {
-            Settings settings = new Settings(settingManager, settingFile);
+            Settings settings = new Settings(settingManager, settingFile, pluginManager);
             settings.ShowDialog();
             SetupLogging();
             SetupHotFolder();
             SetUpdateStrategy();
         }
+
+        /// <summary>
+        /// Selected formatter did change
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void CB_Formatter_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            if (sender is ComboBox box)
+            {
+                if (box.SelectedItem is ComboboxPluginItem item)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Merge this if statement with the enclosing one.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5494671)